docs/blog/2017-09-18-gatsby-modern-static-generation/index.md

```diff
@@ -165,11 +165,11 @@ ops required by such a stack. (There are paid hosting options, of course, but
 that comes with a different set of problems). So I switched to Gatsby.
 
 Another option, if you want to keep Wordpress' Admin UI is to maintain your
-content, is to separate the backend from the frontend.
+content, is to separate the backend from the front end.
```

Comment:
In some of the other changes, you made it `front-end` (which I also prefer), can we use that please?